sdk/core/core-http/package.json

```diff
@@ -44,8 +44,6 @@
     "es/src/**/*.js.map",
     "types/src/**/*.d.ts",
     "types/src/**/*.d.ts.map",
```

Comment:
I think this needs to be updated as well to
```suggestion
    "types/**/*.d.ts",
    "types/**/*.d.ts.map",
```
Since now there is a types/latest and types/3.1 folder

Reply:
yes, just fixed.